save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/FileUtils.kt

```diff
@@ -97,12 +107,16 @@ fun Path.pathNamesTill(stop: Path): List<String> = generateSequence(this, Path::
  * @throws IllegalArgumentException if this path is relative.
  */
 @Throws(IllegalArgumentException::class)
-fun Path.requireIsAbsolute(): Path =
-        apply {
-            require(isAbsolute) {
-                "The path is not absolute: $this"
-            }
-        }
+fun Path.requireIsAbsolute(): Path = apply {
+    require(isAbsolute) {
+        "The path is not absolute: $this"
+    }
+}
+
+actual inline fun <reified C : Any> parseConfig(configPath: okio.Path): C {
```

Comment:
maybe move to `commonNonJsMain`?

Reply:
discussed offline, that it doesn't resolve the code duplication for different targets

Reply:
Discussed offline - decided to leave as is